maud_macros/src/parse.rs

```diff
@@ -148,7 +148,17 @@ impl Parser {
                 }
             },
             // Element
-            TokenTree::Ident(_) => {
+            TokenTree::Ident(ident) => {
+                let ident_string = ident.to_string();
+                // Is this a keyword that's missing a '@'?
+                match ident_string.as_str() {
+                    "if" | "while" | "for" | "match" | "let" => {
+                        ident.span()
+                            .warning(format!("found keyword `{}` - should this be a `@{}`?", ident_string, ident_string))
```

Comment:
Is it possible to use the same argument twice? Something like

```rust
format!("found keyword `{0}` - should this be a `@{0}`?", ident_string)
```

Reply:
It was.